source/items/platearmor.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Plate Armor
+layout: default
+---
+# Plate Armor
+
+<div class="infobox box text-center">
+{% comment %}
```

Comment:
why is this here? take a look at the clipboard page, for example. Also, why is the <br> at the end?